### DISCUSSION TITLE
Iris 3320-L Contact Sensor Tamper

### DISCUSSION THREAD

**Discussion by bryeartem** [Koenkk/zigbee2mqtt#9203] [Devices]
I just moved to Z2M yesterday and in looking at my Iris 3320-L Contact Sensors I see that some show as Tampered, True... as far as I am aware there is no tamper switch on these sensors.  There is no rhyme or reason I can figure out as to which ones show tampered.  Anyone have any insight?  Maybe that value is there from other similar sensors that actually do have a tamper switch?  Anyone found a trick to get them to not show as tampered?

**Comment by dmtalon** [comment]
This doesn't help you one bit, but as a recent user of HA/Z2M, I also see this. While it's not affecting anything that I can tell, it would be nice to reset/turn off the 'tamper' of 'true'
